packages/devextreme/js/__internal/grids/grid_core/columns_resizing_reordering/m_columns_resizing_reordering.ts

```diff
@@ -156,16 +164,28 @@ const SeparatorView = modules.View.inherit({
         return getWidth($element);
       }
     }
-  },
-});
+  }
+}
+
+class ColumnsSeparatorView extends SeparatorView {
+  private _positionChanged: any;
```

Comment:
Let's declare the ```_positionChanged``` as a private method.
Because the method implementation isn't changed in runtime.
And add/remove this method to ```_tablePositionController``` on subscribe/unsubscribe